src/main/java/strategy/DalyOperationStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package strategy;
+
+import java.util.Map;
+import model.OperationType;
+import service.operation.handlers.OperationHandler;
+
+public class DalyOperationStrategyImpl implements DalyOperationStrategy {
```

Comment:
create a package impl and move there
```suggestion
public class OperationStrategyImpl implements OperationStrategy {
```